lib/v8.js

```diff
@@ -15,5 +15,20 @@
 'use strict';
 
 var v8binding = process.binding('v8');
-exports.getHeapStatistics = v8binding.getHeapStatistics;
+var smalloc = require('smalloc');
+
+var data = smalloc.alloc(5, smalloc.Types.Uint32);
```

Comment:
@trevnorris Doesn't this always leak a small amount of memory on exit?  One more thing to add to my valgrind suppression file, I suppose.